src/caf/toolkit/__init__.py

```diff
@@ -3,5 +3,6 @@
 __version__ = _version.get_versions()["version"]
 
 # Alias
-from caf.toolkit import pandas_utils
+from caf.toolkit import pandas_utils, log_helpers
```

Comment:
What am I missing here?
Why are this aliases needed when these imports are already in this location?
I think I put the `pandas_utils` import here too and I can't remember why

Reply:
No idea I thought it was something to do with pytest or tox having issues but I've tested both and they work fine without so I've removed the import line in f7e37786a027d6d7e611ed64dac161737752a1ce